.github/workflows/ci.yml

```diff
@@ -96,15 +96,15 @@ jobs:
 
       - name: Build fluvio
         if: ${{ matrix.binary == 'fluvio' }}
-        run: make build-cli
+        run: cargo xtask build-cli
```

Comment:
this is more verbose than before. can you make alias or reduce boiler plate?

Reply:
IMO this is a small cost to pay for the customization possible with `xtask`. The way cargo aliases work is that the shortest you can get is `cargo <X>`, you can't make a totally independent alias that's not under `cargo`. The main point is to be able to clone the project and without having any external tools (including make) you can just execute your tasks.

Reply:
?? you just replacing one tools (make) with another tool (task) not removing it